src/Sulu/Bundle/MediaBundle/Tests/Functional/Entity/MediaDataProviderRepositoryTest.php

```diff
@@ -145,10 +145,8 @@ private function createCollection($name, $parent = null)
 
     /**
      * @param string $name
-     *
-     * @return TargetGroup
      */
-    public function createTargetGroup($name)
+    public function createTargetGroup($name): TargetGroup
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes